docs/ref/models/querysets.txt

```diff
@@ -2572,23 +2572,25 @@ in the database <database-time-zone-definitions>`.
 hour
 ~~~~
 
-For datetime fields, an exact hour match. Allows chaining additional field
```

Comment:
We remove these annotations after 2 versions, so 1.7 notes don't appear in the development docs at this point.